### DISCUSSION TITLE
Artist Creator: Promote and encourage community members to attend a Multidisciplinary Theatre event!

### DISCUSSION THREAD

**Discussion by ghost** [pair-cg/general-contributions#82] [User Stories]
# 1. Motivating Story
I'm creating an **interdisciplinary theatrical performance**, which will consist of Recorded video and live theatre. I need potential audience members, who are interested in theatre, and historical re-productions to attend this event.
The performance will be based on **historical incidents**, so I want my audience to be aware of it. 
The performance includes **video projection**, So I want to make sure that the audience is aware of the flashing lights, high-volume sound.

# 2. Need:
I need potential audience members to be aware of my performance show.

Information I would like audience made aware of: 

**Title of production**: _The Land you promised_
**Tagline of production**: A historical reproduction on Canada’s racially discriminatory policies. 
**Description of production**: In 1914,376 Indian Immigrants sailed to Vancouver in search of better opportunities. Their arrival posed a challenge to Canada’s anti-Asiatic policies. For two months, they endured unjust treatment, and in the end, all were denied entry. Step into their shoes and witness their resilience. Find out, how the visible minorities shaped Canada’s multiculturalism.  

**Video** to familiarize yourself with the incident - [https://youtu.be/eZyvg_87e6o](url)

(**French version** - [https://youtu.be/CWJAqxJWY1A](url)
(source - Historica Canada)

**Concept/treatment**: On the pattern of reproduction and recreation, the performance will consist of Archival Videos (copyright obtained) as well as re-enactment clips alongside a live theatre consisting of diverse cast members. Production will feature live percussionists from the Mashreq Quartet group. 
**Languages of the performance**: Punjabi, English, Hindi, Urdu, Persian, Cantonese, Malay, Mandarin, and Tamil.
Producing Company: BetterHalf Productions
**Objective of production**: to amplify the contribution of South Asian migration in the inclusivity in Canada, to portray their struggle and movement of resistance against racial policies, and hoping to share these historical achievements with the new generation of diverse Canadians.

**Creative Content information:**
Artistic discipline: multidisciplinary theatre
Genre: Historical
Time-period: 1900s
Original interpreted source material: Vancouver Maritime museum’s exhibition. 

**Event Information:**
Venue: Young Centre for the Performing Arts, (50 Tank House Lane), Toronto, ON
Opening Day: February 13, 2022
Closing Day: February 28, 2022
Start time: 7pm ET
Duration: 129 Minutes
Intermission - Yes (10 minutes)
Surtitles: Yes (in English)

**Ticketing / Registration**
Ticket cost: 

- $55 Regular
- $25 Student/art worker
- Limited number of PWYC tickets

Donation to company: available
Registration: required

# 3. Action:
**Pre Show**

- Targeted social media campaign one month before the show.

- Audience -  Diverse art-lovers, 32-60 year old internet users, art workers, museum goers, theatre visitors. 

- Engage Social media users with contests, giveaway tickets, ‘Do you Know’ posts, encourage them to share  stories through Instagram, Facebook and Twitter. 

 

- Facebook live with cast and crew two weeks before the show.

- Create Opportunity for 10-24 years old to visit rehearsals and engage with shared history. 

- Target South Asian language schools and offer them with 15% discount on tickets + Post show Q and As.  

**During performance run -** 

- Coordinate with cultural radio stations to circulate the number of shows, audience reactions.

- Targeted social media campaign focused on audience/critic reviews. 

- Produce 1-2minute videos to keep flowing the content across channels. (Behind the scenes, interviews,etc)

- Organize a one day panel discussion in the mid of the run, where - invite the experts from South Asian study circle. 

**Post Show**

- Connect with print and TV Media for coverage

 

- Coordinate, pitch and package the show to Art Centers based on other cities if they are interested to organize the performance runs.

# 4. Desired Outcome:
Target audiences (South Asians across North America between the age of 32-70) are reached and made aware of the performance.

# 5. Perspective:
Artist Creator

# 6. Stakeholder Group:
Audience members
Artists/Percussionists 
Search engines 
Arts Organizations
Theatre influencers
Media & Social Media
Listing Websites

# 6. Additional info:
_Here's more information._

**Comment by mariel-marshall** [comment]
@Gursimrandatla I would replicate some of the features of your desired outcome, and put them into your need as they are quite strong and specific. For example: 

Need: 
I need to make target audiences (South Asians across North America between the age of 32-70) aware of my upcoming performance.
